robottelo/helpers.py

```diff
@@ -280,6 +281,20 @@ def add_remote_execution_ssh_key(hostname, key_path=None, **kwargs):
     ssh.add_authorized_key(server_key, hostname=hostname, **kwargs)
 
 
+def add_remote_execution_capsule_to_subnet(hostname, capsule_id=1):
```

Comment:
btw, I personally wouldn't create a helper for every single 'scenario', i don't see a problem in having the following 4 lines of code directly in a test (at least i know what is really going on). But I guess this is very subjective - not a blocker